Tests/Objects/IGTestDiffingSectionController.m

```diff
@@ -43,8 +44,12 @@ - (instancetype)init {
 }
 
 - (CGSize)sectionController:(IGListBindingSectionController *)sectionController sizeForViewModel:(id)viewModel atIndex:(NSInteger)index {
-    const BOOL isString = [viewModel isKindOfClass:[NSString class]];
-    return CGSizeMake([self.collectionContext containerSize].width, isString ? 55 : 30);
+    if ([viewModel isKindOfClass:[IGTestObject class]] && [[(IGTestObject *)viewModel value] isKindOfClass:[IGLayoutTestItem class]]) {
```

Comment:
This looks like a hack to make so many assumptions, can you actually create a simple SectionController that respects on the IGTestObject ? I also think it deserves its own dataSource so that it doesn't affect the existing tests.